packages/loading/lib/messages.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "message": {
+      "text": "You can use middleware to redirect a user when authenticated with <code>redirect('/login')</code>.",
```

Comment:
We may use something like `"text": "You can use middleware to redirect a user when authenticated with `redirect('/login')`."`. Benefits:
- It will work for terminal too (consola detects code to highlight)
- Removes requirement of using `v-html`

Reply:
ohh ok. yeah was wondering how to get rid of the <code> in consola. it might not look as good on the page though. will test it out and add some screens for review